mscore/inspector/inspector_lyric.ui

```diff
@@ -215,12 +215,11 @@
   <customwidget>
    <class>Ms::ResetButton</class>
    <extends>QWidget</extends>
-   <header>inspector/resetButton.h</header>
+   <header location="global">inspector/resetButton.h</header>
```

Comment:
Was this change automatically done by Qt Creator/Designer?

Reply:
yes